apps/sensenet/cypress/integration/content-types/content-types.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+describe('Content types', () => {
+  beforeEach(() => {
+    cy.login()
+    cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl: Cypress.env('repoUrl') } }))
+      .get('div[aria-label="Content Types"]')
```

Comment:
This selector should be changed to a `data-test=Content Types`. This attribute is already added to the drawer component by another contributor so you have to change only the selector here.

Reply:
Done